doc/REAMDE.md

```diff
@@ -0,0 +1,16 @@
+# Openfisca-Core reference documentation
+
+Source of the [openfisca-core reference documentation](http://openfisca.readthedocs.io/).
+
+## Installing
+
+```sh
+    pip install -r requirements.txt
+    pip install sphinx-autobuild
```

Comment:
`sphinx-autobuild` is the most convenient tool to build the doc, but it is not strictly necessary (for instance `readthedocs` doesn't use it).
I could put it in an `optional-requirements.txt` file, would that be better ?